theano/compile/debugmode.py

```diff
@@ -620,37 +620,9 @@ def _optcheck_fgraph(input_specs, output_specs, accept_inplace=False):
     equivalence_tracker = _VariableEquivalenceTracker()
     fgraph = gof.fg.FunctionGraph(orig_inputs, orig_outputs,
                                   features=[equivalence_tracker])
```

Comment:
Thanks. Changed as you asked. I thought this wouldn't work because ```st_fgraph``` had a slightly different implementation for creating the fgraph.